MySQL/test.js

```diff
@@ -0,0 +1,57 @@
+const { getLogger } = require('log4js');
```

Comment:
Please check the comments that i had for PgSQL. The same applies here as well.

Reply:
Fixed all.